batch-1/movie-info/juliat/film.rb

```diff
@@ -0,0 +1,40 @@
+require 'json'
+require 'open-uri'
+
+puts "Pleaes, Enter a film"
+film_name = gets.chomp
+film_name = film_name.split(" ")
+film_name = film_name.join("+")
+req_url = "http://www.omdbapi.com/?t=" + film_name +"&y=&plot=short&r=json"
+request_query = ''
+url = "#{req_url}#{request_query}"
+buffer = open(url).read
+web_content = JSON.parse(buffer)
+
+myfile = File.new("#{web_content["Title"]}.html", "w+")
```

Comment:
Instead of writing HTML inside ruby code, how about this approach:

2 files: film.rb and template.html

In ```template.haml```, we would write something like this:

```
....
<body>
  <h1>{{title}}</h1>
</body>
...
```

We would open ```template.html``` as ```File``` in ruby code and replace ```{{title}}``` with appropriate text using string replace methods. That way we could take the entire html out of ruby code and keep it in a separate template file. Give that a try.